boards/common/particle-mesh/doc.txt

```diff
@@ -0,0 +1,44 @@
+/**
+@defgroup    boards_common_particle-mesh Particle Mesh common
+@ingroup     boards
+@brief       Common support for the Particle Mesh boards family (Xenon, Argon, Boron)
+
+### General information
+
+[Particle Mesh](https://www.particle.io/mesh/) are mesh-ready development kits.
+Depending on the board type (Xenon, Argon, Boron), it provides access to
+multiple communication protocols: BLE, 802.15.4, WiFi, LTE.
+
+### Flash the board
+
+Using the [Particle Mesh debugger](https://docs.particle.io/datasheets/accessories/mesh-accessories/#debugger),
```

Comment:
I think Documentation to flash threw dap link should be included, if someone is to but this board they might not get the DAPlink programmer.

Reply:
I don't understand your comment. The documentation is already explaining how to flash the board using DAPLink (using the debugger accessory)

Reply:
@aabadie sorry I meant DFU-util.

Reply:
Ok, I think DFU programming should be default since it doesn't require an external setup. I'll adapt this accordingly and following @kYc0o's comment.
I'll test this when I'm back with the hardware :)

Reply:
After trying hard, I have to give up with DFU: I can flash but after that the RIOT firmware is never started, the board remains in DFU mode.
I have looked at Zephyr and they don't provide DFU either.

Reply:
I also tried quite a lot during the IETF 104 at Prague and I found that `dfu-util` is unable to flash at address 0, which is the address where the RIOT firmware is linked for. Maybe doing some magic is possible to link it and flash it at another address, maybe after the softdevice firwamre (and thus take advantage of it?)

Reply:
Ok for postponing DFU support then.